src/tito/tagger/main.py

```diff
@@ -243,10 +239,21 @@ def _make_changelog(self):
 
                 write(fd, header)
 
-                for cmd_out in output.split("\n"):
-                    write(fd, "- ")
-                    write(fd, "\n  ".join(textwrap.wrap(cmd_out, 77)))
+                # don't die if this is a new package with no history
+                if self._changelog_msg is not None:
+                    write(fd, self._changelog_msg)
```

Comment:
There is yes, example here: https://github.com/dgoodwin/tito/blob/master/src/tito/cli.py#L448 Just comes out as a standard list.

Indeed you could just pack yours into one msg, just do some magic in the code to add a leading "- " if one isn't present on each --changelog when it's written if you don't mind. That way you can pre-format and pack into a line, and CLI users can just do --changelog "I did a thing." --changelog "I did another thing."